packages/experiments/src/components/PersonaCoin/examples/PersonaCoinSizeAndColor.Example.tsx

```diff
@@ -35,9 +35,9 @@ export class PersonaCoinSizeAndColorExample extends React.Component<{}, {}> {
               <PersonaCoin text="Kevin Jameson" size={100} imageUrl={PersonaTestImages.personMale} presence={4} />
             </Stack>
           </Stack>
-          <Stack gap={personaCoinGap}>
+          <Stack tokens={{ childrenGap: personaCoinGap }}>
             <Text>Custom colors</Text>
-            <Stack horizontal disableShrink gap={personaCoinGap}>
+            <Stack horizontal disableShrink tokens={{ childrenGap: personaCoinGap }}>
```

Comment:
Since these are examples I think it may be better to encourage more scalable code, like defining a centralized token object/function (similar to styles) and then using it or putting it in a theme rather than inlining this object everywhere.